### PR TITLE
Implement project 'start' and natural stop

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/OS-1271) |

#### Main changes

Accessing a project from the project list will:
-  Open a new page saying "Running project 'X'"
- Wait for 3 seconds and run the command provided in the `start` entry from `setup.cfg`

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_miniscreen/app.py [hunk 1]
```diff
@@ -94,25 +94,37 @@ def handler():
...
 
         if self.root.can_perform_action:
             return self.root.perform_action()
 
         if self.root.can_select_row:
-            return self.root.enter_selected_row()
+            self.root.enter_selected_row()
+            self.root.start_project()
+            self.root.wait_project()
+            return self.root.exit_menu()
```
Comment (angusjfw) [inline pt_miniscreen/app.py:110]: i feel like too much of the project page's business has leaked into app & root here.

i think we should leave this button handler as just `enter_selected_row()`, have root push the project page, and then have the ProjectPage do the start & wait (can it do this on init? does it need a thread?). then it could ask root to exit_menu.

also, maybe root should have a method `is_navigation_disabled` instead of `is_project_page`. but I think that part we can leave for now and maybe revisit once we do the project stopping functionality

File: pt_miniscreen/pages/root/projects.py [hunk 2]
```diff
@@ -51,52 +66,87 @@ def from_file(cls, file):
...
+    if "PT_MINISCREEN_SYSTEM" in env:
+        # Allow to take over miniscreen
+        env.pop("PT_MINISCREEN_SYSTEM")
+
+    first_display = get_first_display()
+    if first_display is not None:
+        env["DISPLAY"] = first_display
+
+    # Print output of commands in english
+    env["LANG"] = "en_US.UTF-8"
```
Comment (angusjfw) [inline pt_miniscreen/pages/root/projects.py:81]: is this necessary in this context? we're not parsing the output of people's projects, I think best to leave it in the language they have chosen

**Review by angusjfw — CHANGES_REQUESTED** [review]

**Review by angusjfw — APPROVED** [review]

File: pt_miniscreen/pages/root/projects.py [hunk 3]
```diff
@@ -51,52 +67,94 @@ def from_file(cls, file):
             raise InvalidConfigFile(e)
 
 
+def get_project_environment():
+    env = os.environ.copy()
+
+    if "PT_MINISCREEN_SYSTEM" in env:
```
[the remark this answers is not in the document]
Comment (jcapona) [reply pt_miniscreen/pages/root/projects.py:73]: I get your point, but I think in this case it doesn't really matter, we just want it removed from the project environment
